DataFormats/Scouting/interface/Run3ScoutingMuon.h

```diff
@@ -0,0 +1,305 @@
+  int nValidRecoMuonHits() const { return nValidRecoMuonHits_; }
+  int nRecoMuonChambers() const { return nRecoMuonChambers_; }
+  int nRecoMuonChambersCSCorDT() const { return nRecoMuonChambersCSCorDT_; }
+  int nRecoMuonMatches() const { return nRecoMuonMatches_; }
+  int nRecoMuonMatchedStations() const { return nRecoMuonMatchedStations_; }
+  unsigned int nRecoMuonExpectedMatchedStations() const { return nRecoMuonExpectedMatchedStations_; }
+  unsigned int recoMuonStationMask() const { return recoMuonStationMask_; }
+  int nRecoMuonMatchedRPCLayers() const { return nRecoMuonMatchedRPCLayers_; }
+  unsigned int recoMuonRPClayerMask() const { return recoMuonRPClayerMask_; }
+  int nValidPixelHits() const { return nValidPixelHits_; }
+  int nValidStripHits() const { return nValidStripHits_; }
+  int nPixelLayersWithMeasurement() const { return nPixelLayersWithMeasurement_; }
+  int nTrackerLayersWithMeasurement() const { return nTrackerLayersWithMeasurement_; }
+  float trk_chi2() const { return trk_chi2_; }
+  float trk_ndof() const { return trk_ndof_; }
+  float trk_dxy() const { return trk_dxy_; }
+  float trk_dz() const { return trk_dz_; }
+  float trk_qoverp() const { return trk_qoverp_; }
+  float trk_lambda() const { return trk_lambda_; }
+  float trk_pt() const { return trk_pt_; }
+  float trk_phi() const { return trk_phi_; }
+  float trk_eta() const { return trk_eta_; }
+  float trk_dxyError() const { return trk_dxyError_; }
+  float trk_dzError() const { return trk_dzError_; }
+  float trk_qoverpError() const { return trk_qoverpError_; }
+  float trk_lambdaError() const { return trk_lambdaError_; }
+  float trk_phiError() const { return trk_phiError_; }
+  float trk_dsz() const { return trk_dsz_; }
+  float trk_dszError() const { return trk_dszError_; }
+  //add off-diagonal covariance matrix parameter, the above "Error" variables correspond to the diagonal, enum for Cov matrix (qoverp, lambda, phi, dxy, dsz), see https://github.com/cms-sw/cmssw/blob/CMSSW_11_2_X/DataFormats/TrackReco/src/TrackBase.cc for details
+  float trk_qoverp_lambda_cov() const { return trk_qoverp_lambda_cov_; }
+  float trk_qoverp_phi_cov() const { return trk_qoverp_phi_cov_; }
+  float trk_qoverp_dxy_cov() const { return trk_qoverp_dxy_cov_; }
+  float trk_qoverp_dsz_cov() const { return trk_qoverp_dsz_cov_; }
+  float trk_lambda_phi_cov() const { return trk_lambda_phi_cov_; }
+  float trk_lambda_dxy_cov() const { return trk_lambda_dxy_cov_; }
+  float trk_lambda_dsz_cov() const { return trk_lambda_dsz_cov_; }
+  float trk_phi_dxy_cov() const { return trk_phi_dxy_cov_; }
+  float trk_phi_dsz_cov() const { return trk_phi_dsz_cov_; }
+  float trk_dxy_dsz_cov() const { return trk_dxy_dsz_cov_; }
+  float trk_vx() const { return trk_vx_; }
+  float trk_vy() const { return trk_vy_; }
+  float trk_vz() const { return trk_vz_; }
+  reco::HitPattern trk_hitPattern() const { return trk_hitPattern_; }
```

Comment:
How about
```suggestion
  reco::HitPattern const& trk_hitPattern() const { return trk_hitPattern_; }
```
? (of course depending if `HitPattern` stays or not)